### DISCUSSION TITLE
plotStationary issue with non-numeric-alike variables in data frame: date

### DISCUSSION THREAD

**Discussion by kokoshark13** [bmcclintock/momentuHMM#191] [Q&A] [answered]
Hi! I am trying to use the plotStationary function on a fitHMM output list, but I receive the error 
"Error in Math.data.frame(list(date = 1670657867.59263), 2) : non-numeric-alike variable(s) in data frame : date"

I am attempting to make a visualization that shows the probability of being in state 1 or 2 as a function of covariate SST. I have no issues when using the same HMM list object with plotStates(), timeInStates, plotPR(), etc.
Any help would be greatly appreciated. I can provide the .rds file of the fitHMM output list object if needed.
Thank you

**Comment by bmcclintock** [comment, marked as answer]
I can't help much without a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example#:~:text=In%20computing%2C%20a%20minimal%20reproducible,to%20be%20demonstrated%20and%20reproduced.), but my best guess is that your `date` variable is of an atypical class that is not recognized by `plotStationary`. Functions like `plotStationary` and `plot.momentuHMM` need to be able to calculate a "mean" for all fields in the data (for plotting things as a function of covariates).
